src/styles/index.scss

```diff
@@ -1,3 +1,68 @@
+$days-count: 28, 29, 30, 31;
+$cell-size: 100px;
+$days-of-the-week: (
+  mon: 0,
+  tue: 1,
+  wed: 2,
+  thu: 3,
+  fri: 4,
+  sat: 5,
+  sun: 6,
+);
+
 body {
   margin: 0;
+  height: 100vh;
+  width: 100vw;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+.calendar {
+  display: flex;
+  flex-flow: wrap;
```

Comment:
The `flex-flow` property should be `flex-flow: row wrap;` to ensure that the flex items wrap correctly. Without specifying the direction, it might not behave as expected.